lib/vrl/cli/src/cmd.rs

```diff
@@ -6,37 +6,37 @@ use std::{
     path::PathBuf,
 };
 
-use structopt::StructOpt;
+use clap::Parser;
 use vector_common::TimeZone;
 use vrl::{diagnostic::Formatter, state, Program, Runtime, Target, Value};
 
 #[cfg(feature = "repl")]
 use super::repl;
 use super::Error;
 
-#[derive(Debug, StructOpt)]
-#[structopt(name = "VRL", about = "Vector Remap Language CLI")]
+#[derive(Parser, Debug)]
+#[clap(name = "VRL", about = "Vector Remap Language CLI")]
 pub struct Opts {
     /// The VRL program to execute. The program ".foo = true", for example, sets the event object's
     /// `foo` field to `true`.
-    #[structopt(name = "PROGRAM")]
+    #[clap(name = "PROGRAM")]
     program: Option<String>,
 
     /// The file containing the event object(s) to handle. JSON events should be one per line..
-    #[structopt(short, long = "input", parse(from_os_str))]
+    #[clap(short, long = "input", parse(from_os_str))]
     input_file: Option<PathBuf>,
 
     /// The file containing the VRL program to execute. This can be used instead of `PROGRAM`.
-    #[structopt(short, long = "program", conflicts_with("program"), parse(from_os_str))]
+    #[clap(short, long = "program", conflicts_with("PROGRAM"), parse(from_os_str))]
     program_file: Option<PathBuf>,
 
     /// Print the (modified) event object instead of the result of the final expression. Setting
     /// this flag is equivalent to using `.` as the final expression.
-    #[structopt(short = "o", long)]
+    #[clap(short = 'o', long)]
     print_object: bool,
 
     /// The timezone used to parse dates.
-    #[structopt(short = "tz", long)]
+    #[clap(short = 't', long)]
```

Comment:
**⚠️ Breaking change.**

In clap 3, short is a `char` type.

I chose `t` since it's the first character, but it may be that `z` is more commonly used for timezones.

Strong opinions welcome.

Reply:
I like `-z` just because it would free up `-t`, a more common letter, for something else.